HousingFinanceInterimApi/V1/UseCase/Interfaces/ILoadDirectDebitTransactionsUseCase.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using HousingFinanceInterimApi.V1.Domain;
+using System.Threading.Tasks;
+using HousingFinanceInterimApi.V1.Boundary.Response;
+
+namespace HousingFinanceInterimApi.V1.UseCase.Interfaces
+{
+    public interface ILoadDirectDebitTransactionsUseCase
+    {
+        public Task<StepResponse> ExecuteAsync(DateTime? retroactiveDate = null);
```

Comment:
Why set the default value as null?  Won't that mean it will be set to null if a null argement is provided?  Feels a bit redundant.